torch/_dynamo/variables/functions.py

```diff
@@ -604,13 +604,14 @@ def call_function(
 
 
 class FunctoolsPartialVariable(VariableTracker):
-    def __init__(self, func, args, keywords, **kwargs):
+    def __init__(self, func, args, keywords, original=None, **kwargs):
         super().__init__(**kwargs)
         self.func = func
         assert isinstance(args, list)
         self.args = args
         assert isinstance(keywords, dict)
         self.keywords = keywords
+        self.original = original
```

Comment:
🤔 This is weird. If original is set, I assume this is because you wrapped a functools.partial? But then how do you know the args/keywords??

Reply:
oh because you read them out of the fields 🤔

Reply:
This is it, so it should be self contained.